RELEASE/scripts/autoscend/paths/wildfire.ash

```diff
@@ -430,8 +430,8 @@ boolean LX_wildfire_water()
 	//use water in a variety of ways to reduce fire levels. putting it in pre-adv is problematic since we need to spend adventures here
 	//individual location watering first
 	
-	//for stone wool. needed at level 11 but we acquire it early using [Baa'baa'bu'ran]. no qualifiers since this is always needed
-	if(LX_wildfire_hose($location[The Hidden Temple])) return true;
+	//for stone wool. needed at level 11 but we acquire it early using [Baa'baa'bu'ran]. Skip if we've somehow already progressed past that stage
+	if(internalQuestStatus("questL11Worship") < 3 && LX_wildfire_hose($location[The Hidden Temple])) return true;
```

Comment:
Does ash support short circuit conditionals?

Reply:
Yes.
although personally I am not a fan as it makes it somewhat unclear. I prefer to do
```
if(internalQuestStatus("questL11Worship") < 3)
{
     if(LX_wildfire_hose($location[The Hidden Temple])) return true;
}
```

Reply:
```suggestion
```
last commit added a duplicate copy of this